### PR TITLE
Glossary localisation

### PR DESCRIPTION
**Pull request by jorge8989**
https://pfeffermind.atlassian.net/browse/GRM-65

### CODE REVIEW

File: _includes/glossary.html [hunk 1]
```diff
@@ -10,14 +10,26 @@ <h1 class="h2 drawer-header"><icon class="icon-book"></icon> Glossary </h1>
...
-    <ul class="js-glossary-list list-unstyled" accordion="glossary-accordion">
-      {% for term in site.data.terms %}
-      <li class="glossary-item">
-        <h2 class="h5 glossary-term">{{term[0]}}</h2>
-        <button><span class="sr-only">Toggle for {{term[0]}}</span></button>
-        <p class="glossary-definition accordion-content">{{term[1]}}</p>
-      </li>
-      {% endfor %}
-    </ul>
+    {% if site.lang == "de" %}
```
Comment (messutied-pf) [inline _includes/glossary.html]: we might have more than one language at some point, better do this in a way that it simply uses the current lang, something like this:

```ruby
for term in site.data["terms_#{site.lang}"]
```

must be possible

File: _data/terms_en.yml [hunk 2]
```diff
@@ -74,6 +74,3 @@
  "Variance floor": "During the reconciliation process, only variances between reported numbers that exceed a minimum dollar amount are investigated by the Independent Administrator."
  "Wet gas": "Natural gas that hasn’t been treated to remove liquid hydrocarbons or other nonhydrocarbons that make the gas unmarketable."
```
Comment (messutied-pf) [inline _data/terms_en.yml]: can we move the terms files to the _i18n folder?

File: _includes/glossary.html (hunk 1 above)
Comment (jorge8989) [inline _includes/glossary.html]: Yeah, i tried that yesterday. It doesn't work. Coming from rails, i also thought that was possible. And i know it's just ruby, but for some reason it doesn't as expected. If i try to interpolate site.lang, it returns nothing. I tried site.data['terms_'+site.lang],  site.data["terms_#{site.lang}"], site.data["terms_#{site_lang_variable}"], site.data[some_variable], maybe there's a Jekyll way for doing this, I'll search.

Comment (jorge8989) [inline _includes/glossary.html]: I just asked a question on Stackoverflow.

**Pull request merged**